impala/tests/conftest.py

```diff
@@ -0,0 +1,105 @@
+# (C) Datadog, Inc. 2022-present
+# All rights reserved
+# Licensed under a 3-clause BSD style license (see LICENSE)
+import os
+from unittest import mock
+
+import pytest
+
+from datadog_checks.dev import docker_run, get_docker_hostname, get_here
+from datadog_checks.dev.conditions import CheckDockerLogs
+from datadog_checks.impala import ImpalaCheck
+
+
+def pytest_configure(config):
+    config.addinivalue_line(
+        "markers", "metrics_file(directory, name): The path of the fixture file to use for this test"
+    )
+
+
+@pytest.fixture(scope="session")
+def dd_environment():
+    compose_file = os.path.join(get_here(), "compose", "docker-compose.yaml")
+    with docker_run(
+        compose_file=compose_file,
+        conditions=[
+            CheckDockerLogs(
+                identifier=compose_file,
+                patterns=[
+                    "Connected to metastore.",  # Statestore
+                    "Impala has started.",  # Daemon
+                    "CatalogService started",  # Catalog
+                ],
+                matches='all',
+                wait=5,
+                attempts=20,
+            )
+        ],
+        endpoints=[
+            f"http://{get_docker_hostname()}:25000/metrics_prometheus",  # Daemon
+            f"http://{get_docker_hostname()}:25010/metrics_prometheus",  # Statestore
+            f"http://{get_docker_hostname()}:25020/metrics_prometheus",  # Catalog
+        ],
+    ):
+        yield {
+            "openmetrics_endpoint": f"http://{get_docker_hostname()}:25000/metrics_prometheus",
```

Comment:
The e2e env worked for me 🎉 Is there a reason why we don't use a config with all instances for the e2e env?

Reply:
The reason is that I wrote this for the POC and never updated it 😅 Updating it :)

Reply:
Great, thanks!